config.toml

```diff
@@ -20,13 +21,22 @@ user = "" # REQUIRED
 pass = "" # REQUIRED
 cert = "" # REQUIRED
 
+[eth_rpc]
+server = "" # REQUIRED
+port = "" # REQUIRED
```

Comment:
ethereum node has no authentication?

Reply:
there is no secure connection like btcd, and expose read-only commands by rpc,  listen on 127.0.0.1 may be safest way
```
Please note, offering an API over the HTTP (rpc) or WebSocket (ws) interfaces will give 
everyone access to the APIs who can access this interface (DApps, browser tabs, etc). Be 
careful which APIs you enable. By default Geth enables all APIs over the IPC (ipc) interface 
and only the db, eth, net and web3 APIs over the HTTP and WebSocket interfaces.

refer:https: //github.com/ethereum/go-ethereum/wiki/Management-APIs
```

Reply:
OK, add this to the README in the section about setting up ethd